requirements/base.txt

```diff
@@ -90,6 +90,8 @@ dnspython==2.1.0
     # via email-validator
 email-validator==1.1.3
     # via flask-appbuilder
+exceptiongroup==1.2.0
+    # via cattrs
```

Comment:
I've executed `pip-compile-multi --no-upgrade` and got no such changes, are you based on the latest master on this PR? anyway I think these should be removed

Reply:
I don't know where this comes from, I did not change something else?
I made a re-sync with the master and run it again, it still appears in my output?

If I am looking to the package "cattrs" it seems it is a mandatory requirement and it should be there?
I can also manual remove it if it is not good?

https://github.com/apache/superset/blob/master/requirements/base.txt
```
cattrs==23.2.1
    # via requests-cache
```

if I check this package local it says that exceptiongroup is a mandatory sub-dependency?
```
....pip show cattrs
Name: cattrs
Version: 23.2.1
Summary: Composable complex class support for attrs and dataclasses.
Home-page:
Author:
Author-email: Tin Tvrtkovic <tinchester@gmail.com>
License: MIT
Location: /home/starnol/.local/lib/python3.10/site-packages
Requires: attrs, exceptiongroup, typing-extensions
Required-by:
```

shall i still remove it?

Reply:
I am using the Ubuntu subsystem (WSL2) in windows with the normal installed python which you can get with apt-get

```
... $ python3 --version
Python 3.10.12

.... ~$ pip --version
pip 24.0 from /home/..../.local/lib/python3.10/site-packages/pip (python 3.10)

// no updates available
~$ sudo apt-get upgrade -y
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Calculating upgrade... Done
The following packages have been kept back:
  ubuntu-advantage-tools
0 upgraded, 0 newly installed, 0 to remove and 1 not upgraded.

// no update for python3
~$ sudo apt-get install python3
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
python3 is already the newest version (3.10.6-1~22.04).
python3 set to manually installed.
0 upgraded, 0 newly installed, 0 to remove and 1 not upgraded.
```

Reply:
Thank you, I thinks this is correct